### PR TITLE
Update docker-entrypoint.sh

### PR DESCRIPTION
**Pull request by tarrenj**
Remove depreciated `-a` file operator in favor of `-e`.

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -40,7 +40,7 @@ if [[ "$1" == apache2* ]] || [ "$1" == php-fpm ]; then
 		group="$(id -g)"
 	fi
 
-	if ! [ -e index.php -a -e wp-includes/version.php ]; then
+	if ! [ -e index.php -e wp-includes/version.php ]; then
```
Comment (tianon) [inline docker-entrypoint.sh]: This isn't quite right:

```console
$ [ -e xyz -e abc ]
-bash: [: too many arguments
```

What this needs to be updated to instead is something like:

```bash
	if [ ! -e index.php ] && [ ! -e wp-includes/version.php ]; then
```

Comment (tarrenj) [reply docker-entrypoint.sh]: ACK

Comment (tarrenj) [reply docker-entrypoint.sh]: Resolved.  Thanks for the review.

**Comment by tianon** [comment]
Thanks!

**Pull request merged**